docs/design/run.md

```diff
@@ -0,0 +1,268 @@
+# Introducing JJ run
```

Comment:
Perhaps this should just be called "`jj run` design" or "Outline of `jj run` design"?

Reply:
I like the current title, so I'm going to keep it. A version of the above would be approriate for $WORK.